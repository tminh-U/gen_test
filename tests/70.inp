9
14 16 9 11 8 16 20 10 10 
10
2 6
3 5
7 9
4 7
5 6
6 9
7 9
6 9
9 9
1 4
