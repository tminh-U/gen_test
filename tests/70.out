12
5
5
12
4
7
5
7
2
7
